core/datainput.gms

```diff
@@ -1461,6 +1461,40 @@ pm_emifac(ttot,regi,"seliqfos","fehos","tdfoshos","co2") = p_ef_dem(regi,"fehos"
 pm_emifac(ttot,regi,"seliqfos","fepet","tdfospet","co2") = p_ef_dem(regi,"fepet") / (sm_c_2_co2*1000*sm_EJ_2_TWa); !! GtC/TWa
 pm_emifac(ttot,regi,"seliqfos","fedie","tdfosdie","co2") = p_ef_dem(regi,"fedie") / (sm_c_2_co2*1000*sm_EJ_2_TWa); !! GtC/TWa
 
+***------ Read in emission factors for process emissions in chemicals sector---
+*' calculated using IEA data on feedstocks flows and UNFCCC data on chem sector process emissions
+***TO DO: remove the "*" by modif the input file (fegas instead of gases, etc.)
```

Comment:
I wanted to remove the `*` here: `parameter f_nechem_emissionFactors(ttot,all_regi,*)` because I don't know if using that trick is proper coding for us. This means introducing some changes in data preparation in `mrremind` that I was postponing for the final cleaning before merging. Or can I just leave it like that?